backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/types/CardVerificationResultModel.kt

```diff
@@ -4,5 +4,6 @@ import java.time.Instant
 
 data class CardVerificationResultModel(
     val valid: Boolean,
+    val extendable: Boolean = false,
```

Comment:
Hmm... I think that verifiers should not be able to retrieve this information.
Possibly, we should create a new endpoint that uses the totp secret as authentication.

Reply:
I assumed it's not a sensitive value but more technical, and it's not handled in the app when requested from the verifier

Reply:
I mean, I agree that for Koblenz, it does not matter because (afaik) the self-service portal would also reveal this information... Feel free to resolve it.